web/src/components/DandisetsPage.vue

```diff
@@ -204,7 +207,7 @@ export default defineComponent({
         sortDir.value *= -1;
       } else {
         sortOption.value = index;
-        sortDir.value = 1;
+        sortDir.value = -1;
```

Comment:
Nothing wrong here, once I decoded what this function is doing... but boy is it a confusing function! No need to do anything in this PR, but I feel like this function could use a comment explaining what it's doing.